docs/software/comanage-troubleshooting-guide.md

```diff
@@ -0,0 +1,77 @@
+COmanage Troubleshooting Guide
+====
+Introduction
+----
+The purpose of this guide is to be a resource for COmanage administrators to find solutions to commonly encountered problems.
+
+
+COManage Troubleshooting Items
+----
+This section contains some of the issues you may encounter when interacting with the OSG COmanage.
+
+Problem list:
+
+ - [Tiger-Dex Unable to match User to COmanage CO Person identity](#tiger-dex-unable-to-match-user-to-comanage-co-person-identity)
+
+###Tiger-Dex Unable to match User to COmanage CO Person identity
+
+Under certain circumstances Dex may not be able to match a user to their COmanage identity,
+ possibly due to incorrect and/or missing organization identities/identifiers.
+
+####Symptoms
+
+Symptoms include:
+
+ - Tiger-Dex credentials not including some or all of a users group permissions.
+ - Not being able to connect to the Tiger cluster.
```

Comment:
```suggestion
 - Being unable to connect to the Tiger cluster.
```

It'd be worth adding an example error message here. Inability to connect will manifest differently than failure to authenticate/authorize (this problem is the latter, anyway)

Reply:
Do you happen to still have an error message you and/or Huijin encountered when you were dealing with this issue that we could add to the doc or is a general "error message from Tiger referring to a failure to authenticate/authorize" good enough?